client/src/components/Podiums/Podiums.js

```diff
@@ -75,6 +83,29 @@ const Podiums = ({ match }) => {
               />
             </Grid>
           ))}
+          {competition.events.length !== competition.podiums.length && (
+            <Grid item key={`events_not_completed`}>
```

Comment:
The `key` prop is necessary only when rendering a list (usually with `Array#map`) and that's not that we do here =)